README.md

```diff
@@ -308,15 +319,15 @@ Joplin uses and renders a Github-flavoured Markdown with a few variations and ad
 
 # Custom CSS
 
-Rendered markdown can be customized by placing a userstyle file in the profile directory `~/.config/joplin-desktop/userstyle.css` (This path might be different on your device - check at the top of the Config screen for the exact path). This file supports standard CSS syntax. Joplin ***must*** be restarted for the new css to be applied, please ensure that Joplin is not closing to the tray, but is actually exiting. Note that this file is used for both displaying the notes and printing the notes. Be aware how the CSS may look printed (for example, printing white text over a black background is usually not wanted).
+Rendered markdown can be customized by placing a userstyle file in the profile directory `~/.config/joplin-desktop/userstyle.css` (This path might be different on your device - check at the top of the Options screen (Tools > Options)) for the exact path). This file supports standard CSS syntax. Joplin ***must*** be restarted for the new css to be applied, please ensure that Joplin is not closing to the tray, but is actually exiting. Note that this file is used for both displaying the notes and printing the notes. Be aware how the CSS may look printed (for example, printing white text over a black background is usually not wanted).
```

Comment:
Please revert this. On macOS there is no `Tools > Options`.

Reply:
I wasn't aware there was a toolbar difference between macOS and Windows/Linux. I might have to dig out an ancient Mac mini that is lying around somewhere and come back to this after using the Mac version to be more familiar with the differences.

Reply:
On Linux/Windows you go to `Tools > Options` and on macOS to `Joplin > Preferences...`.

This is the reason why we used a rather general term and wording. Maybe we should use the term settings and explain what the term means (the first time it occurs in the README). 

Not sure what the best way is to handle this. I'm open to suggestions.

Reply:
Yes @tessus is correct, we use relatively generic terms because some parts of the doc apply to macOS or Windows, or sometimes even mobile/desktop/cli.